### PR TITLE
Listing all answers in a table

### PR DESCRIPTION
**Pull request by HristijanSlavkoski**

### CODE REVIEW

File: project/First-Project/src/main/java/first/project/repository/AnswerRepository.java [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
 
+import java.util.ArrayList;
+
 import first.project.model.Answer;
 
 @Repository
 public interface AnswerRepository extends JpaRepository<Answer, Long>
 {
+    @Query(value = "SELECT * FROM Answer a WHERE a.questionid=?1", nativeQuery = true)
+    public ArrayList<Answer> getAllAnswersById(Integer id);
```
Comment (dejannastevski-armedia) [inline project/First-Project/src/main/java/first/project/repository/AnswerRepository.java]: Minor: Change the method name to be more descriptive like "getAllAnswersByQuestionId" or something like that.

File: project/First-Project/src/main/java/first/project/service/AnswerService.java [hunk 2]
```diff
@@ -11,4 +13,6 @@
     boolean checkEmail(String email);
 
     Answer validateAndSave(AnswerDTO answerDTO) throws InvalidAnswerException;
+
+    ArrayList<Answer> getAllAnswersById(Integer id);
```
Comment (dejannastevski-armedia) [inline project/First-Project/src/main/java/first/project/service/AnswerService.java]: Same here

File: project/First-Project/src/main/java/first/project/repository/AnswerRepository.java (hunk 1 above)
Comment (aleksandar-bujaroski-armedia) [inline project/First-Project/src/main/java/first/project/repository/AnswerRepository.java]: Rename this to getAllAnswerForQuestion

File: project/First-Project/src/main/java/first/project/service/AnswerService.java (hunk 2 above)
Comment (aleksandar-bujaroski-armedia) [inline project/First-Project/src/main/java/first/project/service/AnswerService.java]: Rename this to getAllAnswerForQuestion

**Review by aleksandar-bujaroski-armedia — CHANGES_REQUESTED** [review]

**Review by dejannastevski-armedia — APPROVED** [review]

**Review by aleksandar-bujaroski-armedia — APPROVED** [review]

**Pull request merged**